site/app/views/AutoGradingView.php

```diff
@@ -387,7 +387,7 @@ public function showTAResults(TaGradedGradeable $ta_graded_gradeable, bool $regr
         $gradeable_id = $gradeable->getId();
         $id = $this->core->getUser()->getId();
         if($gradeable->isTeamAssignment()){
-            $id = $this->core->getQueries()->getTeamByGradeableAndUser($gradeable_id, $id)->getId();
+            $id = $this->core->getTeamIds($id, $gradeable_id);
```

Comment:
To follow the proper MVC format, it would be better to get the data in the controller and pass the raw data into the view rather than calling a function on the controller. The view should not call a controller. The controller should only use the one function for one view.